production/helm/loki/values.yaml

```diff
@@ -678,6 +678,8 @@ write:
   # -- Tolerations for write pods
   tolerations: []
   persistence:
+    # -- Enable StatefulSetAutoDeletePVC feature
+    enableStatefulSetAutoDeletePVC: true
```

Comment:
I'm nervous about this as the default for write. I think we should default to `false`, as I want to err on the side of not losing WALs when someone doesn't intentionally set this value.

Reply:
I'm agree with you, let's go `false` by default for write. And maybe `false` by default too for singleBinary ?
What about backend ? `true` seems relevant imho.